Only warn on package repositories that can't be resolved

Do not throw exception, which prevents server from starting.  For example, if an exception is thrown when trying to resolve the experimental repository.

```
2018-08-30T11:01:00.106-05:00 [APP/PROC/WEB/0] [OUT] Caused by: 2018-08-30T11:01:00.106-05:00 [APP/PROC/WEB/0] [OUT] Caused by: java.io.IOException: Server returned HTTP response code: 503 for URL: http://skipper-repository.cfapps.io/repository/experimental/index.yml
2018-08-30T11:01:00.106-05:00 [APP/PROC/WEB/0] [OUT] at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1894) ~[na:1.8.0_181]
2018-08-30T11:01:00.106-05:00 [APP/PROC/WEB/0] [OUT] at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492) ~[na:1.8.0_181]
2018-08-30T11:01:00.106-05:00 [APP/PROC/WEB/0] [OUT] at org.springframework.core.io.UrlResource.getInputStream(UrlResource.java:169) ~[spring-core-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
2018-08-30T11:01:00.106-05:00 [APP/PROC/WEB/0] [OUT] at org.springframework.cloud.skipper.server.service.PackageMetadataService.downloadPackageMetadata(PackageMetadataService.java:194) ~[spring-cloud-skipper-server-core-1.0.7.RELEASE.jar!/:1.0.7.RELEASE]```

--- a/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/service/PackageMetadataService.java
+++ b/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/service/PackageMetadataService.java
@@ -208,8 +208,8 @@ public class PackageMetadataService implements ResourceLoaderAware {
 						}
 					}
 				}
-				catch (IOException e) {
-					throw new SkipperException("Could not process package file from " + packageRepository.getName(), e);
+				catch (Exception e) {
+					logger.warn("Could not process package file from " + packageRepository.getName(), e.getMessage());
 				}
 			}
 		}

